sycl/include/CL/sycl/ordered_queue.hpp

```diff
@@ -26,46 +26,56 @@ class context;
 class device;
 class ordered_queue {
 public:
+  [[deprecated("Replaced by the in_order property in queue")]]
```

Comment:
I think it easier to apply attribute to the class.
Also this attribute requires C++14 support, so it should be pre-processed. I'm not sure if I know the right condition to enable it.
Here is some reference: https://godbolt.org/z/eNr__M